src/styles/themes/default.js

```diff
@@ -55,8 +55,10 @@ const darkTheme = {
     placeholderText: colors.greenIcons,
     heroCard: colors.blue,
     uploadPreviewActivityIndicator: colors.greenHighlightBackground,
+    checkBox: colors.green,
 };
 
+// DEPRECATED THEME, Delete after successful rebranding.
```

Comment:
Which issues are blocking the removal of the oldTheme? Do we have a list or tracker

Reply:
I don't think should remove these from the oldTheme. But yeah we can remove the comment. As per https://github.com/Expensify/App/pull/12944#issuecomment-1324284696 we will need this.

Reply:
Resolving this as I deleted this [here](https://github.com/Expensify/App/pull/13083) to speed things up.